### ISSUE TITLE
Remove the usages accessing sumo API using username/password

### ISSUE THREAD

**Issue by lei-sumo** [SumoLogic/sumologic-collector-chef-cookbook#99]
As we are deprecating username/password authentication on api and collection registration, we need to make sure no such usage inside this cookbook.

**Comment by zl4bv** [comment]
Since this would be a breaking change we should consider releasing it as version 2.0. It would be a good opportunity to remove the installer resource too, which was replaced with the default resource.

**Comment by majormoses** [comment]
agreed, I think we are honestly due for a major version bump. I would take a pr (will not be released to the supermarket for a bit as I am trying to also do some other stuff in as well and have better testing) for this.
